boot/zephyr/CMakeLists.txt

```diff
@@ -63,7 +63,15 @@ set(NRF_DIR "${MCUBOOT_DIR}/ext/nrf")
 
 if(CONFIG_BOOT_USE_NRF_CC310_BL)
 set(NRFXLIB_DIR ${ZEPHYR_BASE}/../nrfxlib)
-assert_exists(NRFXLIB_DIR)
+if(NOT EXISTS ${NRFXLIB_DIR})
+  message(FATAL_ERROR "
+    ------------------------------------------------------------------------
+    No such file or directory: ${NRFXLIB_DIR}
+    Current configuration enables nRF CC310 crypto accelerator hardware
+    by `BOOT_USE_NRF_CC310_BL` option. Please follow `ext/nrf/README.md` guide
```

Comment:
"**The** current configuration enables ..."; "... accelerator hardware **with** ..."; and I would use `CONFIG_BOOT_USE_NRF_CC310_BL` instead